ci/cloudbuild/builds/demo-install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This build script serves two main purposes:
+#
+# 1. It demonstrates to users how to install `google-cloud-cpp`. We will
+#    extract part of this script into user-facing markdown documentation.
+# 2. It verifies that the installed artifacts work by compiling and running the
+#    quickstart programs against the installed artifacts.
+
+set -eu
+
+source "$(dirname "$0")/../../lib/init.sh"
+source module ci/cloudbuild/builds/lib/cmake.sh
+source module ci/etc/quickstart-config.sh
+source module ci/lib/io.sh
+
+## [START packaging.md]
+
+# Pick a location to install the artifacts, e.g., `/usr/local` or `/opt`
+export PREFIX="${HOME}/google-cloud-cpp-installed"
```

Comment:
nit: export?  Do we need to define this variable at all?

Reply:
Export not needed and removed. This build script runs as a non-root user, so I can't install in `/usr/local`. I needed to change it to some dir that's writable.

I tried using `DESTDIR`, which almost works, but I could never get it to actually work correctly.

Let me know if you have another idea about how I could avoid the `-DCMAKE_INSTALL_PREFIX`

Reply:
Showing the `-DCMAKE_INSTALL_PREFIX` LGTM, I just did not want to imply that an export was required.